wasm/src/lib.rs

```diff
@@ -77,7 +71,7 @@ impl Execute for data_model::query::QueryBox {
     /// Executes the given query on the host environment
     fn execute(&self) -> Self::Result {
         #[cfg(not(test))]
-        use host::execute_query as host_execute_query;
+        use host::host__execute_query as host_execute_query;
```

Comment:
I like this convention of FFI functions having a `Struct__method` and `mod__fn` convention.